cmd/create.go

```diff
@@ -127,6 +127,10 @@ A full list of component types that can be deployed is available using: 'odo com
 			checkError(err, fmt.Sprintf("failed to push component: %v", componentName))
 
 			fmt.Printf("Component '%s' was created.\n", componentName)
+		} else if catalogType == "template" {
```

Comment:
what is someones does something that is from the catalog and combines it with `--git` or `--local`  argument?
We should probably refactor this big `if` `else` ...  to something nicer, and less prone to errors.